nexus/src/authz/api_resources.rs

```diff
@@ -191,12 +191,69 @@ impl AuthorizedResource for Fleet {
     }
 }
 
-#[derive(Clone, Copy, Debug)]
-pub struct GlobalImageList {
-    fleet: Fleet,
+/// ConsoleSessionList is a synthetic resource used for modeling who has access
+/// to create sessions.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
```

Comment:
It seems like something about the way this was merged caused this stuff to show up as part of your change.  That's weird.